pretests/docker_test_images/docker_test_images.py

```diff
@@ -0,0 +1,80 @@
+r"""
+Summary
+---------
+
+Pre-testing pulling and listing of test image(s) into sysinfo results
+
+Operational Summary
+----------------------
+
+#. Parse the default test image into FQIN format
+#. Pull the default, and any configured ``extra_fqins_csv`` images
```

Comment:
> So if I understand correctly, this introduces a new extra_fqins_csv. Why not just use the existing preserve_fqins, and not do error checks on pulls?

I considered that actually, but the two have subtly different meanings. My intention here is/was to allow for an optional ``docker_test_images.ini`` with this option.  Either way, the ``preserve_fqins`` list is advisory during cleanup, so it must support items that possibly don't exist. Whereas, anything pulled by ``docker_test_images`` __must__ exist, which allows the framework to fail early on some repo. server or network problem.

Also, with it this way, it's possible to copy this test into ``intratests/`` and have it guarantee an image exists between every test.